igor-web/src/main/groovy/com/netflix/spinnaker/igor/jenkins/client/model/Build.groovy

```diff
@@ -56,6 +58,11 @@ class Build {
      */
     @ElementList(required = false, name = "action", inline = true)
     List<TestResults> testResults
+
+    GenericBuild genericBuild(String jobName) {
```

Comment:
Is there a reason why you don't have Build implement the GenericBuild interface, rather than write these conversion methods? There's nothing wrong with the conversion methods, it just feels a little strange